tools/generate-function-bindings/scripts/generate.ts

```diff
@@ -66,7 +67,7 @@ async function fetchMetadata() {
     .map(([name, signature]) => renderStaticMethod(name, signature));
 
   const fnClass = t.exportNamedDeclaration(
-    t.classDeclaration(t.identifier("Fn"), null, t.classBody(staticMethods))
+    t.classDeclaration(t.identifier("FnGenerated"), null, t.classBody(staticMethods))
```

Comment:
I don't get this rename, but I'll see how this develops further. Leaving a comment here for my own memory

Reply:
Did this get solved? Or shall I explain?

Reply:
Please explain for me

Reply:
`Fn` in `terraform-functions.ts` extends this class (to allow overriding some functions, refer to that file for why) – As we extend this class, JSII requires us to also export it and this requires us to have a unique name.